float32 confidence
string text
string command
string scope
float32 position
float32 position_confidence
int32 id
